src/itkOMEZarrNGFFImageIO.cxx

```diff
@@ -401,6 +402,75 @@ OMEZarrNGFFImageIO::ReadArrayMetadata(std::string path, std::string driver)
   }
 }
 
+ImageIORegion
+OMEZarrNGFFImageIO::ConfigureTensorstoreIORegion(const ImageIORegion & ioRegion) const
+{
+
+  // Set up IO region to match known store dimensions
+  itkAssertOrThrowMacro(m_StoreAxes.size() == store.rank(), "Detected mismatch in axis count and store rank");
+  ImageIORegion storeRegion(store.rank());
+  itkAssertOrThrowMacro(storeRegion.GetImageDimension(), store.rank());
+  auto storeAxes = this->GetAxesInStoreOrder();
+
+  for (size_t storeIndex = 0; storeIndex < store.rank(); ++storeIndex)
```

Comment:
If axis order is not x->y->z (or is it z->y->x?), the direction matrix will not be identity. We should probably handle that case now, too.

Reply:
We currently assume that the remote zarr store axes will appear in the order "z", "y", "x". For instance:
- `t,c,z,y,x`: observed to be more common, channels are slower moving than spatial axes
- `t,z,y,x,c`: observed to be less common, channels are fastest moving axis

I am in favor of improving handling later. Will enter an issue to track.

Reply:
Tracked in https://github.com/InsightSoftwareConsortium/ITKIOOMEZarrNGFF/issues/45.